src/Compilers/CSharp/Test/Emit/PDB/PDBLambdaTests.cs

```diff
@@ -89,115 +89,22 @@ public static int Main()
     }
 
     static public int M(int p)
-    {
+    {|method:{|closure:{
```

Comment:
>{| [](start = 4, length = 2)

Nit: Can we use `<|` and `|>` instead of `{|` and `|}`? The curly braces are a harder to visually distinguish from `{`, `}`.